### PR TITLE
WIP: Medtronic 600 documentation

### PR DESCRIPTION
**Pull request by pazaan**
Documentation of more of the 600-series protocol packets, and a sequence diagram.

### CODE REVIEW

**Comment by pazaan** [comment]
[<M> Medtronic 6 - Remaining Work](https://trello.com/c/B4p2NziK/139-m-medtronic-6-remaining-work)

File: lib/drivers/medtronic600/docs/packetStructure.md [hunk 1]
```diff
@@ -16,11 +17,12 @@ All other messages are broken down to USB packets before being sent on the HID i
 
 ### MiniMed Message
 A **MiniMed Message** is broken up into [USB Packet](#usb-packet)s.
+
 ![MiniMed Message](images/svg/MiniMedMessage.svg)
 
 | Field     | Bytes | Data Type  |   Comments   |
 |-----------|:-----:|:----------:|--------------|
-| Header    | 2     | Bytes      | `0x5103`     |
+| Header    | 2     | Bytes      | `0x5103` (`0x03` is an NGP device)     |
```
Comment (gniezen) [inline lib/drivers/medtronic600/docs/packetStructure.md]: Maybe mention what an NGP device is, as it's the first mention of it in this doc.

File: lib/drivers/medtronic600/docs/packetStructure.md [hunk 2]
```diff
@@ -45,22 +47,58 @@ A **MiniMed Message** is broken up into [USB Packet](#usb-packet)s.
...
+A **Read Info Request** is contained in the payload of a [MiniMed Message](#minimed-message) where the Minimed Operation is `0x14` (READ_INFO). It has no message payload.
+
+### Read Info Response
+A **Read Info Response** is contained in the payload of a [MiniMed Message](#minimed-message) where the Minimed Operation is `0x14` (READ_INFO).
+
+![Read Info Response](images/svg/ReadInfoResponse.svg)
+
+| Field     | Bytes | Data Type  |   Comments   |
+|-----------|:-----:|:----------:|--------------|
+| CNL MAC   | 8     | UInt64BE   | ZigBee MAC Address of the Contour Next Link |
```
Comment (gniezen) [inline lib/drivers/medtronic600/docs/packetStructure.md]: Is it actually ZigBee, or just IEEE 802.15.4? I would imagine for it to be ZigBee it needs to interoperate with other Zigbee devices, and I doubt that Medtronic would go to that effort.

Comment (pazaan) [inline lib/drivers/medtronic600/docs/packetStructure.md]: Good point. I thought those two were interchangeable, but as you point out, it's only IEEE 802.15.4 for this.

**Pull request merged**